doc/source/getting-started/index.rst

```diff
@@ -1,6 +1,6 @@
-########
-Overview
-########
+###############
+Getting started
+###############
 
 The PyAnsys project exposes Ansys technologies via libraries in the 
```

Comment:
Here it is "PyAnsys project" with a lowercase "p"--I think this is what we should use.

Reply:
I agree. Let's go with this naming.